packages/waku/src/router/client.ts

```diff
@@ -186,6 +211,7 @@ export function Link({
   unstable_prefetchOnEnter,
   unstable_prefetchOnView,
   unstable_startTransition,
+  scroll = true,
```

Comment:
Hmm, I don't prefer "default true". Can it be the same default behavior with `newPath` like useRouter?